Dockerfile.jsonschema

```diff
@@ -0,0 +1,6 @@
+# 3.18.2
+FROM alpine@sha256:25fad2a32ad1f6f510e528448ae1ec69a28ef81916a004d3629874104f8a7f70
+RUN apk add --update protoc protobuf-dev go git
```

Comment:
Should we be pinning versions for these tools?

Reply:
I think so, yeah -- IIRC we've seen `protoc` make codegen changes between releases before, so pinning here may prevent rude CI discrepancies 🙂